docs/puml/text2diagram.puml

```diff
@@ -91,8 +91,8 @@ class discopy.rigid.Ob #back:lightblue;line:black {}
 class discopy.monoidal.Ty #back:lightblue;line:black {}
 class discopy.monoidal.Diagram #back:lightblue;line:black {}
 class discopy.rigid.Diagram #back:lightblue;line:black {}
-class discopy.biclosed.Diagram #back:lightblue;line:black {}
-class discopy.biclosed.Box #back:lightblue;line:black {}
+class discopy.grammar.categorial.Diagram #back:lightblue;line:black {}
```

Comment:
Note that we'll have to manually create the image files for those diagrams, there is no automatic generation.